tokenLists/arbitrum.tokenlist.json

```diff
@@ -58,10 +58,10 @@
         {
             "chainId": 42161,
             "address": "0x3F56e0c36d275367b8C502090EDF38289b3dEa0d",
-            "symbol": "MAI",
-            "name": "miMATIC",
             "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/MAI.png"
+            "name": "MAI",
+            "symbol": "MIMATIC",
```

Comment:
name should be Mai StableCoin, symbol is MAI
https://arbiscan.io/token/0x3F56e0c36d275367b8C502090EDF38289b3dEa0d